### PR TITLE
Fix the issue where getting a error when removing role from permissio…

### PR DESCRIPTION
**Pull request by NipunaMarcus**
…n which is not belong to the user's roles

### CODE REVIEW

File: apps/portal/js/dashboard-settings.js [hunk 1]
```diff
@@ -70,20 +70,20 @@ $(function () {
             properties.buttons = [
                 {
                     addClass: 'btn btn-primary', text: 'Ok', onClick: function ($noty) {
-                        $noty.close();
-                        if (ok) {
-                            ok();
-                        }
+                    $noty.close();
```
Comment (sinthuja) [inline apps/portal/js/dashboard-settings.js]: Formatting issue.

File: apps/portal/js/dashboard-settings.js [hunk 2]
```diff
@@ -70,20 +70,20 @@ $(function () {
...
                     }
+                }
                 },
                 {
                     addClass: 'btn btn-danger', text: 'Cancel', onClick: function ($noty) {
-                        $noty.close();
-                        if (cancel) {
-                            cancel();
-                        }
+                    $noty.close();
```
Comment (sinthuja) [inline apps/portal/js/dashboard-settings.js]: Formatting issue - Indentation.

File: apps/portal/store/carbon.super/gadget/usa-social/index.xml [hunk 3]
```diff
@@ -55,8 +55,8 @@ var draw = function(data) {
     document.getElementById("chart").innerHTML = "";
     
     var margin = {top: 20, right: 50, bottom: 30, left: 40},
-        width = window.document.body.scrollWidth - margin.left - margin.right,
-        height = window.document.body.scrollHeight - margin.top - margin.bottom;
+        width = window.document.body.scrollWidth - margin.left - margin.right - 20,
+        height = window.document.body.scrollHeight - margin.top - margin.bottom -60;
```
Comment (sinthuja) [inline apps/portal/store/carbon.super/gadget/usa-social/index.xml]: Formatting issue, need to have space for '-'.

File: apps/portal/js/dashboard-settings.js (hunk 1 above)
Comment (NipunaMarcus) [inline apps/portal/js/dashboard-settings.js]: Fixed with the PR https://github.com/wso2/carbon-dashboards/pull/173

File: apps/portal/js/dashboard-settings.js (hunk 2 above)
Comment (NipunaMarcus) [inline apps/portal/js/dashboard-settings.js]: Fixed with the PR https://github.com/wso2/carbon-dashboards/pull/173

File: apps/portal/store/carbon.super/gadget/usa-social/index.xml (hunk 3 above)
Comment (NipunaMarcus) [inline apps/portal/store/carbon.super/gadget/usa-social/index.xml]: Fixed with the PR https://github.com/wso2/carbon-dashboards/pull/173

**Pull request merged**